src/theme/src/default-theme/typography/headings.js

```diff
@@ -81,6 +81,7 @@ export default {
     fontSize: '12px',
     fontWeight: 600,
     lineHeight: '16px',
+    letterSpacing: '0',
```

Comment:
Actually, I wonder if we want this to be `normal` (the default value of this property). 

> The normal letter spacing for the current font. Unlike a value of `0`, this keyword allows the user agent to alter the space between characters in order to justify text.

https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing

Reply:
I would also use `'0'`, since size `300` also has it: https://github.com/segmentio/evergreen/blob/ec4552dee7561178f0a0291dd3cd34040a5a578d/src/theme/src/default-theme/typography/headings.js#L75.

Reply:
Fair enough!